elementary/monitor/cli.py

```diff
@@ -207,9 +205,10 @@ def report(ctx, days_back, config_dir, profiles_dir, update_dbt_package, profile
     track_cli_start(anonymous_tracking, 'monitor-report', get_cli_properties(), ctx.command.name)
     try:
         data_monitoring = DataMonitoring(config, update_dbt_package)
-        success = data_monitoring.generate_report(days_back=days_back, test_runs_amount=executions_limit,
-                                                  file_path=file_path,
+        success = data_monitoring.generate_report(anonymous_tracking.anonymous_user_id, days_back=days_back,
```

Comment:
please be specific when passing an argument to a method:
```suggestion
        success = data_monitoring.generate_report(user_id=anonymous_tracking.anonymous_user_id, days_back=days_back,
```
This is a better practice IMO - it is safer + it makes it really easy to understand what each argument is used for.